### ISSUE TITLE
Bobril build prompts user to resolve version conflicts

### ISSUE THREAD

**Issue by krausest** [krausest/js-framework-benchmark#142]
Bobril now and then prompts the user to resolve conflicts like (or #91):

```
warning bobril-build > systemjs > es6-module-loader@0.17.11: This project has been deprecated for "npm install es-module-loader" based on the newer loader spec.
info Unable to find a suitable version for "uglify-js", please choose one by typing one of the numbers below:
  1) "uglify-js@2.*" which resolved to "2.8.8"
  2) "uglify-js@~2.3" which resolved to "2.3.6"
Answer?:
```
@Bobris 
I'd prefer that bb does not prompt the user. How can we make the bobril build more stable?

**Comment by Bobris** [comment]
Bobril-build now by default runs yarn install at start. You can disable this feature by adding "dependencies": "disable"
into "bobril" section in package.json.
We don't see this problem because all users of bobril-build I know have it installed globaly. 
Right now bobril-build installs with 2.8.8 version of uglify-js and everything works. Also you would need to answer these questions only once (choose always highest version is ok)- after that it will be stored inside package.json and not ask you again. But in your case it is better to switch off this feature anyway.

**Comment by krausest** [comment]
I tried in 2812a9adc87c60e0a1648e6198c525940502415f, but I'm still getting the question. What am I doing wrong?

**Comment by Bobris** [comment]
This feature was introduced in 0.58.0 version of Bobril-build and you have fixed 0.57.0. But please directly update to latest which is now 0.67.0.

**Comment by krausest** [comment]
That works - thanks.
